tests/ha/priority_fencing_delay.pm

```diff
@@ -60,6 +63,14 @@ sub run {
         assert_script_run "crm configure clone promotable-1 stateful-1 meta promotable=true";
         assert_script_run "crm resource param stonith-sbd set pcmk_delay_max 15";
         assert_script_run "crm configure property priority-fencing-delay=30";
+        # Workaround for bsc#1244437
+        if (get_var("WORKAROUND_BSC1244437")) {
+            assert_script_run "crm configure location force_primary promotable-1 100: $hostname";
```

Comment:
```suggestion
            assert_script_run 'crm configure location force_primary promotable-1 100: ' . get_hostname;
```

as only used once, you do not need a variable.

Reply:
Updated the code :+1: